apps/graphql-playground/src/components/GqlPlayground.tsx

```diff
@@ -83,17 +88,41 @@ function GqlPlayground(props: GqlPlaygroundProps) {
         },
       ];
 
-  const settings = { "editor.theme": "light" };
+  const settings = { 'editor.theme': 'light' };
+
+  useEffect(() => {
+    const unsubscribe = store.subscribe(() => {
+      const query: string = getQuery(store.getState());
+      const result = query.toLowerCase().includes('collection');
```

Comment:
Unfortunately, that's the only way I managed to pull the data from [graphql-playground-react](https://github.com/graphql/graphql-playground/tree/main/packages/graphql-playground-react)

They don't expose the codemirror instance not provide another way to get the query value.